go.mod

```diff
@@ -73,6 +59,7 @@ require (
 )
 
 replace (
+	github.com/syncthing/syncthing => github.com/rberrelleza/syncthing v1.2.2-gen1
```

Comment:
why is this?

Reply:
Forgot to mention on the PR. the syncthing repo generates some source code when building the binary (like the UI's HTML and CSS assets). I forked the repo,checked out the 1.2.2 branch,  generated the assets file, commited it and pushed it to my fork so we can directly import it in our binary.